app/models/concerns/date_accessible.rb

```diff
@@ -4,7 +4,6 @@ module DateAccessible
   TAX_YEAR = Date.new(Rails.configuration.statefile_current_tax_year)
 
   included do
-    private
```

Comment:
what's the significance of making these not private methods?

Reply:
The methods from the module are class methods, so they can't be private, and I believe this line has no effect